src/EnergyPlus/MixedAir.cc

```diff
@@ -996,12 +957,12 @@ void GetOutsideAirSysInputs(EnergyPlusData &state)
     lAlphaBlanks.dimension(MaxAlphas, true);
     lNumericBlanks.dimension(MaxNums, true);
 
-    CurrentModuleObject = CurrentModuleObjects(static_cast<int>(CMO::ControllerList));
+    std::string_view CurrentModuleObject = CurrentModuleObjects(static_cast<int>(CMO::ControllerList));
```

Comment:
Maybe I should have left this as a string since this gets modified again at line 1015. Or I could use 2 different string names? and set these as constexpr?